### PR TITLE
Update JS api parser dependency

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Updated ts-node and typescript dependency to resolve API parsing issue for JS packages

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
Fixes https://github.com/Azure/azure-sdk-tools/issues/5277

File: tools/apiview/parsers/js-api-parser/package.json [hunk 1]
```diff
@@ -9,8 +9,9 @@
     "js-tokens": "^6.0.0"
   },
   "devDependencies": {
-    "ts-node": "^8.10.2",
-    "typescript": "^3.9.7"
+    "@types/node": "^12.20.55",
```
Comment (deyaaeldeen) [inline tools/apiview/parsers/js-api-parser/package.json:12]: nitpick: Could we go higher? to 14 for example?

**Review by deyaaeldeen — APPROVED** [review]

File: tools/apiview/parsers/js-api-parser/package.json (hunk 1 above)
Comment (praveenkuttappan) [reply tools/apiview/parsers/js-api-parser/package.json]: Done.

**Pull request merged**